package/python-kiwi-spec-template

```diff
@@ -121,12 +121,14 @@ Provides:       kiwi-image-tbz-requires = %{version}-%{release}
 Obsoletes:      kiwi-image-tbz-requires < %{version}-%{release}
 %if "%{_vendor}" != "debbuild"
 Provides:       kiwi-image:tbz
+Provides:       kiwi-image:enclave
```

Comment:
This should also be under the same conditional as what provides eif_build.

Reply:
ok, done

Reply:
once we have eif_build for other distros it's easier to increase the scope. Yes that's better